### DISCUSSION TITLE
Ground Items - Show above value

### DISCUSSION THREAD

**Discussion by CodePanter** [runelite/runelite#18456] [Ideas]
Hello!
I would really like the opposite of the 'hide under value' setting; that being something that will show an item even if its on your blacklist, whenever the value exceeds a certain value. The idea is that if for example you blacklist sharks because they are worthless; but then get a drop of 1000 sharks; you might want it because thats a lot of money.

**Comment by raiyni** [comment]
you can use quantity conditions in blacklist/highlight https://github.com/runelite/runelite/wiki/Ground-Items

**Reply by CodePanter** [reply]
That would work for the specific example I gave, however:
1: I would have to do that for all items I blacklist; whereas, a blanket 'if its total value is more than X, I want to see it' would be much more useful.
2: Relying on quantity also heavily relies on me knowing what is valuable, and what becomes valuable. By basing it on value, I wouldn't have to keep up with (for example with mixology) guams suddenly doubling in value. They would just pop up because of their value.
